### ISSUE TITLE
Urgent:  libpam-python update, cannot ssh login

### ISSUE THREAD

**Issue by hiliang-cmu** [picoCTF/picoCTF#478]
Recent update of `libpam-python` to version `1.0.6-1.1+deb10u1build0.18.04.1` seems to cause an issue whereby the typical system paths for python modules no longer are available at this point https://github.com/picoCTF/picoCTF/blob/ba3829ba141b1d2479d2487aa1f7df68f2572936/ansible/pico-shell/files/pam_auth.py#L9

This results in ssh login failing for all users except any system ones decoupled from platform auth e.g. `ubuntu`

`sys.path` output shows that `/usr/local/lib/python2.7/dist-packages` and `/usr/lib/python2.7/dist-packages` are no longer there.

A quick hack is to add them manually before `requests` is imported

```
import sys
sys.path += ['/usr/local/lib/python2.7/dist-packages', '/usr/lib/python2.7/dist-packages']
```

A longterm/proper solution is required.

The aforementioned `libpam-python` package update is stated to be an Ubuntu fake sync, so it is not possible to rollback to the most recent previously working version.

**Comment by hiliang-cmu** [comment]
Note: rolling back to the last `libpam-python` version that appears to be available, `1.0.6-1` *does* seem to fix the issue. Unfortunately there is a CVE fix mentioned in the changelogs since then, so this may not be desirable

**Comment by jrolli** [comment]
As a note, you only need to add `/usr/lib/python2.7/dist-packages` (location of the requests library) to fix the issue.  Looking at the CVE and associated patch, I think we are fine as an attacker would already have to have a write primitive with root privileges to take advantage of pico's specific usecase so your hotfix is OK.
